src/BuildScriptGenerator/Hugo/HugoPlatformInstaller.cs

```diff
@@ -32,6 +32,10 @@ public virtual string GetInstallerScriptSnippet(string version)
             var snippet = new StringBuilder();
             snippet
                 .AppendLine()
+                .AppendLine("IMAGES_DIR=\"/opt/tmp/images\"")
+                .AppendLine("if grep -q jamstack \"/opt/oryx/.imagetype\"; then")
```

Comment:
Nit: Please consider adding a comment explaining the goal of this. Minor comment, feel free to ignore if you have no other changes to push